### PR TITLE
Event recovery

### PR DESCRIPTION
**Pull request by davidakachaos**
As reported in #36 this will restore validator correctly after switching to and from Event mode.

Ping @berestovskyy to look this over a bit before merge.

### CODE REVIEW

File: src/release.js [hunk 1]
```diff
@@ -32,8 +32,11 @@
  *************************************************************************/
 
 /**
- * WV RELEASE INFORMATION
- */
+* WV RELEASE INFORMATION
+*/
+// Fixes error building the I18n
+var WV_NAME = "WME Validator";
+var WV_SHORTNAME = "Validator";
```
Comment (berestovskyy) [inline src/release.js]: There is a commit https://github.com/WMEValidator/i18n/commit/1545a70a16cd577b1a18348417b888b188ed3777 removing those constants.

**Comment by berestovskyy** [comment]
Otherwise looks fine to me.

File: src/release.js (hunk 1 above)
Comment (davidakachaos) [reply src/release.js]: I didn't see that, oops.
So need to remove this again....

Comment (berestovskyy) [reply src/release.js]: No problem, I merged without this commit ;)

Comment (davidakachaos) [reply src/release.js]: Great! Thanks 😄

**Pull request closed without merging**